docs/source/models/causes/maternal_disorders/gbd_2021_mncnh/maternal_hemorrhage.rst

```diff
@@ -13,9 +13,107 @@ GBD 2021 Modeling Strategy
 Cause Hierarchy
 +++++++++++++++
 
+- All causes (c_294) [level 0]
+
+  - Communicable, maternal, neonatal, and nutritional diseases (c_295)
+
+    - Maternal disorders and neonatal disorders (c_962)
+
+      - Maternal disorders (c_366)
+
+        - Maternal hemorrhage (c_367)
+
+          - Maternal hemorrhage with less than 1 liter blood loss (s_180)
+
+          - Maternal hemorrhage with greater than 1 liter blood loss (s_181)
+
+          - Mild anemia due to maternal hemorrhage (s_182)
+
+          - Moderate anemia due to maternal hemorrhage (s_183)
+
+          - Severe anemia due to maternal hemorrhage (s_184)
+
+*Maternal hemorrhage (c_367)* ia a most detailed cause, at level 4 of the GBD hierarchy. 
+It has five sequelae, detailed in the following table:
+
+.. list-table:: Sequelae of maternal hemorrhage
+    :header-rows: 1
+    :widths: 2 1 5 5
+
+    * - Sequela
+      - GBD ID
+      - Health state and disability weight
+      - Notes
+    * - Maternal hemorrhage with less than 1 liter blood loss
+      - s_180
+      - abdominopelvic problem, moderate 
+
+        DW: 0.324 (0.22–0.442) 
```

Comment:
double check this, it is weird that moderate is higher weight than severe

Reply:
Whoops, typo, thank you for catching!